.scripts/build.yml

```diff
@@ -4,6 +4,9 @@ steps:
       versionSpec: "16.x"
     displayName: "Use Node 16"
   - script: |
-      npm install
-      npm run build
+      npm install -g @microsoft/rush  
+    displayName: "Install Rush"
+  - script: |
+      rush update
+      rush build
```

Comment:
lets use rush scriopts which are designed to be used in CI

```
node common/scripts/install-run-rush.js update
node common/scripts/install-run-rush.js build
```

Reply:
updated.